### DISCUSSION TITLE
Status updates

### DISCUSSION THREAD

**Discussion by creewick** [creewick/MontereyRainmeter#60] [Announcements]
## Changes
- Re-created the clock widget in 2 styles: analog and digital
- Created localisation structure, currently supported languages:
  - Russian
  - Ukrainian
  - English
  - Deutsch
  - Polish
  - Vietnamese
- Re-created light and dark themes with transparency support (Windows 11 only, but I hope to find a working solution for Windows 7+)
- Context menus
- Some attempts for re-creation of the calendar widget. It's kinda ugly and aren't transparent yet

<img width="375" alt="Снимок экрана 2023-04-07 в 22 37 48" src="https://user-images.githubusercontent.com/32510906/230645211-5036e3b1-c0ad-43da-8450-2e1251081d5a.png">

## About the project
The whole thing is going to be a .NET Core 6.0 application. It will support Windows 7 SP1 and newer.
The source code will be private until some stable beta releases.
I'm using WPFUI library to allow transparency and modern context menus.

**Comment by WonTheDev** [comment]
Nice! Keep up the good work!
